From: Kill Your TV <killyourtv@mail.i2p>
Date: Fri, 12 Oct 2012 18:08:57 +0000
Subject: sound paths

---
 src/backend/SoundManager.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/backend/SoundManager.cpp b/src/backend/SoundManager.cpp
index 7aa42cf..b1a830c 100755
--- a/src/backend/SoundManager.cpp
+++ b/src/backend/SoundManager.cpp
@@ -89,12 +89,12 @@ void CSoundManager::reInit()
 		settings.endGroup();
 
 		settings.beginGroup("SoundFilePath");
-			mSoundFileUser_go_Online	=settings.value("User_go_Online","./sounds/Startup.wav").toString();
+			mSoundFileUser_go_Online	=settings.value("User_go_Online","/usr/share/i2p-messenger/sounds/Startup.wav").toString();
 			mSoundFileUser_go_Offline	=settings.value("User_go_Offline","")			.toString();
 			mSoundFileFileSend_Finished	=settings.value("FileSend_Finished","")			.toString();
-			mSoundFileFileRecive_Incoming	=settings.value("FileRecive_Incoming","./sounds/File.wav").toString();
+			mSoundFileFileRecive_Incoming	=settings.value("FileRecive_Incoming","/usr/share/i2p-messenger/sounds/File.wav").toString();
 			mSoundFileFileRecive_Finished	=settings.value("FileRecive_Finished","")		.toString();
-			mSoundFileNewChatMessage	=settings.value("NewChatMessage","./sounds/Notify.wav")	.toString();
+			mSoundFileNewChatMessage	=settings.value("NewChatMessage","/usr/share/i2p-messenger/sounds/Notify.wav")	.toString();
 		settings.endGroup();
 	settings.endGroup();
 }
