From: Kill Your TV <killyourtv@i2pmail.org>
Date: Mon, 9 Mar 2015 22:48:19 +0000
Subject: Hardening flags

---
 I2P-Messenger.pro         | 4 ++--
 I2P-Messenger_release.pro | 4 +++-
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/I2P-Messenger.pro b/I2P-Messenger.pro
index 6e206a1..dfe0ebb 100755
--- a/I2P-Messenger.pro
+++ b/I2P-Messenger.pro
@@ -5,8 +5,8 @@ CONFIG += qt \
 QT += network xml
 TEMPLATE = app
 
-QMAKE_CXXFLAGS+= -std=c++11 -Wall
-QMAKE_LFLAGS  += -std=c++11
+QMAKE_CXXFLAGS+= -std=c++11 -Wall $(shell dpkg-buildflags --get CXXFLAGS)
+QMAKE_LFLAGS  += -std=c++11 $(shell dpkg-buildflags --get LDFLAGS)
 
 DEPENDPATH += ". \
 	      src/gui/ \
diff --git a/I2P-Messenger_release.pro b/I2P-Messenger_release.pro
index 0f87b23..ab0a456 100755
--- a/I2P-Messenger_release.pro
+++ b/I2P-Messenger_release.pro
@@ -6,8 +6,10 @@ CONFIG += qt \
 QT += network xml
 TEMPLATE = app
 
+CXXFLAGS += $(shell dpkg-buildflags --get CXXFLAGS)
+
 QMAKE_CXXFLAGS+= -std=c++11 -Wall
-QMAKE_LFLAGS  += -std=c++11
+QMAKE_LFLAGS  += -std=c++11 $(shell dpkg-buildflags --get LDFLAGS)
 
 DEPENDPATH +=  ". \
 	      src/gui/ \
